### PR TITLE
add withdraw button to states dropdown

### PR DESCRIPTION
**Pull request by dreamfall**

### CODE REVIEW

File: app/assets/stylesheets/admin/base.scss [hunk 1]
```diff
@@ -1146,3 +1146,27 @@ table .ellipsis {
...
+    text-align: center;
+    background: none;
+    padding: 0;
+    border: none;
+    display: block;
+    padding: 3px 20px;
+    clear: both;
+    font-weight: normal;
+    line-height: 1.42857143;
+    color: #333333;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Color literals like `#333333` should only be used in variable declarations; they should be referred to via variable everywhere else.

File: app/assets/stylesheets/admin/base.scss [hunk 2]
```diff
@@ -1146,3 +1146,27 @@ table .ellipsis {
...
+    white-space: nowrap;
+    width: 100%;
+    text-align: left;
+
+    &:hover {
+      text-decoration: none;
+      color: #262626;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Color literals like `#262626` should only be used in variable declarations; they should be referred to via variable everywhere else.

File: app/assets/stylesheets/admin/base.scss [hunk 3]
```diff
@@ -1146,3 +1146,27 @@ table .ellipsis {
...
+      background-color: #f5f5f5;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Color literals like `#f5f5f5` should only be used in variable declarations; they should be referred to via variable everywhere else.

File: app/assets/stylesheets/admin/base.scss [hunk 4]
```diff
@@ -1146,3 +1146,27 @@ table .ellipsis {
...
 }
+
+.button_to {
+  .as-link {
+    text-align: center;
+    background: none;
+    padding: 0;
+    border: none;
+    display: block;
+    padding: 3px 20px;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Property `padding` already defined on line 1154

File: app/assets/stylesheets/admin/base.scss [hunk 5]
```diff
@@ -1146,3 +1146,27 @@ table .ellipsis {
...
+    clear: both;
+    font-weight: normal;
+    line-height: 1.42857143;
+    color: #333333;
+    white-space: nowrap;
+    width: 100%;
+    text-align: left;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Property `text-align` already defined on line 1152

File: app/assets/stylesheets/admin/base.scss [hunk 6]
```diff
@@ -1146,3 +1146,27 @@ table .ellipsis {
     }
   }
 }
+
+.button_to {
+  .as-link {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Rule set contains (13/10) properties

File: app/assets/stylesheets/admin/base.scss [hunk 7]
```diff
@@ -1146,3 +1146,27 @@ table .ellipsis {
...
+    text-align: center;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Properties should be ordered display, width, padding, padding, clear, border, background, color, font-weight, line-height, text-align, text-align, white-space

File: app/assets/stylesheets/admin/base.scss [hunk 8]
```diff
@@ -1146,3 +1146,27 @@ table .ellipsis {
...
+    clear: both;
+    font-weight: normal;
+    line-height: 1.42857143;
+    color: #333333;
+    white-space: nowrap;
+    width: 100%;
+    text-align: left;
+
+    &:hover {
+      text-decoration: none;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Properties should be ordered background-color, color, text-decoration

File: app/assets/stylesheets/admin/base.scss [hunk 9]
```diff
@@ -1146,3 +1146,27 @@ table .ellipsis {
     }
   }
 }
+
+.button_to {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Selector `button_to` should be written in lowercase with hyphens

File: spec/features/admin/form_answers/withdrawal_spec.rb [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+describe "Admin withdraws application", js: true do
+  let(:admin) { create(:admin) }
+
+  let(:user) do
+    FactoryGirl.create :user, :completed_profile, first_name: "Test User john"
+  end
+
+  let(:form_answer) do
+    FactoryGirl.create :form_answer, :innovation,
+                                     user: user,
```
Comment (matthewford) [inline spec/features/admin/form_answers/withdrawal_spec.rb]: Align the parameters of a method call if they span more than one line.

File: app/assets/stylesheets/admin/base.scss [hunk 11]
```diff
@@ -1146,3 +1145,25 @@ table .ellipsis {
...
+    clear: both;
+    font-weight: normal;
+    line-height: 1.42857143;
+    color: #333333;
+    white-space: nowrap;
+    width: 100%;
+    text-align: left;
+
+    &:hover {
+      background-color: #f5f5f5;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Color literals like `#f5f5f5` should only be used in variable declarations; they should be referred to via variable everywhere else.

File: app/assets/stylesheets/admin/base.scss [hunk 12]
```diff
@@ -1146,3 +1145,25 @@ table .ellipsis {
...
+      color: #262626;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Color literals like `#262626` should only be used in variable declarations; they should be referred to via variable everywhere else.

File: app/assets/stylesheets/admin/base.scss [hunk 13]
```diff
@@ -1146,3 +1145,25 @@ table .ellipsis {
     }
   }
 }
+
+.button_to {
+  .as-link {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Rule set contains (11/10) properties

File: app/assets/stylesheets/admin/base.scss [hunk 14]
```diff
@@ -1146,3 +1145,25 @@ table .ellipsis {
...
+    background: none;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Properties should be ordered display, width, padding, clear, border, background, color, font-weight, line-height, text-align, white-space

File: app/models/assessor.rb [hunk 15]
```diff
@@ -50,6 +50,7 @@ def self.available_for(category)
 
   def applications_assigned_to_as(roles = ["regular", "lead"])
     FormAnswer.for_award_type(assigned_categories_as(roles))
+              .where("state NOT IN (?)", FormAnswerStatusFiltering.internal_states("withdrawn"))
```
Comment (matthewford) [inline app/models/assessor.rb]: Place the . on the previous line, together with the method call receiver.

**Comment by RuslanHamidullin** [comment]
Good to merge :+1:

**Pull request merged**